bootstrap/lumen.php

```diff
@@ -15,7 +15,7 @@
     realpath(__DIR__.'/../')
 );
 
-// $app->withFacades();
+$app->withFacades();
```

Comment:
Did you say this needs testing?

Reply:
Sorta - the branch is green so its tested :)